src/pushsource/_impl/model/erratum.py

```diff
@@ -265,6 +265,14 @@ class ErratumPushItem(PushItem):
     :type: list[ErratumPackageCollection]
     """
 
+    container_list = attr.ib(
+        type=list, default=attr.Factory(frozenlist), converter=frozenlist
+    )
+    """A list of containers ssociated with the advisory.
```

Comment:
```suggestion
    """A list of container images associated with the advisory.
```

Also, could you please be a little more specific here?  Are they image pull specs?  By digest, by tag or both?

Also, does the order of this list matter?  If it doesn't, can we please canonicalize the data by sorting the list - this will ensure that an erratum with container_list of [A, B] vs a container_list of [B, A] are considered to be the same object.